src/App.tsx

```diff
@@ -1,12 +1,18 @@
 import React from 'react';
 import './App.scss';
 
-// import todos from './api/todos';
-// import users from './api/users';
+import todos from './api/todos';
+import users from './api/users';
+import { TodoList } from './components/TodoList';
+
+const preparedTodos: PreparedTodo[] = todos.map(todo => ({
+  ...todo,
+  user: users.find(user => user.id === todo.userId) || null,
+}));
 
 const App: React.FC = () => (
   <div className="App">
-    <h1>Static list of todos</h1>
+    <TodoList preparedTodos={preparedTodos} />
```

Comment:
the simpler the better

```suggestion
    <TodoList todos={preparedTodos} />
```